05/26/2025 08:51:21 PM - INFO - Banco de dados não encontrado. Criando...
05/26/2025 08:51:21 PM - INFO - Banco de dados criado com sucesso.
05/26/2025 08:51:21 PM - INFO - Banco de dados já existe.
05/26/2025 08:51:21 PM - INFO - Iniciando sign-up
05/26/2025 08:51:22 PM - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/Users/marina.oliveira/Documents/FIAP/PROJETO FINAL TC/TechChallengeFase1/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
05/26/2025 08:51:22 PM - ERROR - Usuário admin já existe.
05/26/2025 08:51:37 PM - INFO - Banco de dados já existe.
05/26/2025 08:51:37 PM - INFO - Banco de dados já existe.
05/26/2025 08:51:37 PM - INFO - Iniciando sign-up
05/26/2025 08:51:37 PM - INFO - Usuário admin2 cadastrado com sucesso.
05/26/2025 08:55:35 PM - INFO - Banco de dados já existe.
05/26/2025 08:55:35 PM - INFO - Banco de dados já existe.
05/26/2025 08:55:35 PM - INFO - Iniciando sign-up
05/26/2025 08:55:35 PM - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/Users/marina.oliveira/Documents/FIAP/PROJETO FINAL TC/TechChallengeFase1/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
05/26/2025 08:55:35 PM - ERROR - Usuário admin2 já existe.
05/26/2025 08:55:46 PM - INFO - Banco de dados já existe.
05/26/2025 08:55:46 PM - INFO - Banco de dados já existe.
05/26/2025 08:55:46 PM - INFO - Iniciando sign-up
05/26/2025 08:55:47 PM - INFO - Usuário user cadastrado com sucesso.
05/26/2025 08:58:43 PM - WARNING - (trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/Users/marina.oliveira/Documents/FIAP/PROJETO FINAL TC/TechChallengeFase1/env/lib/python3.12/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
05/26/2025 09:11:50 PM - INFO - Iniciando scraping de processamento.
05/26/2025 09:11:51 PM - ERROR - Erro ao acessar http://vitibrasil.cnpuv.embrapa.br/index.php?ano=1992&opcao=opt_03&subopcao=subopt_01: HTTPConnectionPool(host='vitibrasil.cnpuv.embrapa.br', port=80): Max retries exceeded with url: /
05/26/2025 09:11:51 PM - ERROR - Erro ao capturar dados do banco: 'Product'
05/26/2025 09:14:12 PM - INFO - Iniciando scraping de processamento.
05/26/2025 09:14:12 PM - INFO - Acesso ao site bem-sucedido.
05/26/2025 09:14:12 PM - INFO - Dados do site coletados com sucesso
05/26/2025 09:14:36 PM - INFO - Iniciando scraping de processamento.
05/26/2025 09:14:36 PM - ERROR - Erro ao acessar http://vitibrasil.cnpuv.embrapa.br/index.php?ano=1992&opcao=opt_03&subopcao=subopt_03: HTTPConnectionPool(host='vitibrasil.cnpuv.embrapa.br', port=80): Max retries exceeded with url: /
05/26/2025 09:14:36 PM - ERROR - Erro ao capturar dados do banco: 'Product'
05/26/2025 09:14:42 PM - INFO - Iniciando scraping de processamento.
05/26/2025 09:14:42 PM - ERROR - Erro ao acessar http://vitibrasil.cnpuv.embrapa.br/index.php?ano=1992&opcao=opt_03&subopcao=subopt_03: HTTPConnectionPool(host='vitibrasil.cnpuv.embrapa.br', port=80): Max retries exceeded with url: /
05/26/2025 09:14:42 PM - ERROR - Erro ao capturar dados do banco: 'Product'
05/26/2025 09:15:02 PM - INFO - Iniciando scraping de processamento.
05/26/2025 09:15:02 PM - ERROR - Erro ao acessar http://vitibrasil.cnpuv.embrapa.br/index.php?ano=1992&opcao=opt_03&subopcao=subopt_01: HTTPConnectionPool(host='vitibrasil.cnpuv.embrapa.br', port=80): Max retries exceeded with url: /
05/26/2025 09:15:02 PM - ERROR - Erro ao capturar dados do banco: 'Product'
05/26/2025 09:15:03 PM - INFO - Iniciando scraping de processamento.
05/26/2025 09:15:03 PM - ERROR - Erro ao acessar http://vitibrasil.cnpuv.embrapa.br/index.php?ano=1992&opcao=opt_03&subopcao=subopt_01: HTTPConnectionPool(host='vitibrasil.cnpuv.embrapa.br', port=80): Max retries exceeded with url: /
05/26/2025 09:15:03 PM - ERROR - Erro ao capturar dados do banco: 'Product'
